x-pack/plugins/ml/public/application/timeseriesexplorer/components/entity_control/entity_control.tsx

```diff
@@ -129,13 +224,146 @@ export class EntityControl extends Component<EntityControlProps, EntityControlSt
           defaultMessage: 'Enter value',
         })}
         singleSelection={{ asPlainText: true }}
+        onCreateOption={this.onManualInput}
+        customOptionText={i18n.translate('xpack.ml.timeSeriesExplorer.setManualInputHelperText', {
+          defaultMessage:
+            'No records found. You can still proceed with any value from the source index to view the single metric.',
```

Comment:
> No records found. You can still proceed with any value from the source index to view the single metric.

In my opinion, the second sentence is not necessary since it seems to just tell you to keep doing what you were already doing. Some alternative options for the first sentence are as follows:

> This value (or entity?) does not have records with anomalies in the selected time range.
> No records with anomalies were found for this entity in the selected time range.

Reply:
Or even `No anomalous records found.` ?

Reply:
> No anomalous records found.

Yes, I disregarded that option since it felt like it was saying the records themselves were anomalous, but for the sake of brevity I think it's okay.